ext/vscode/src/utils/azureDevCli.ts

```diff
@@ -171,26 +164,20 @@ function normalize(env: NodeJS.ProcessEnv): Environment {
 }
 
 function azdNotInstalledMsg(): string {
-    return localize("azure-dev.utils.azd.notInstalled", "Azure Developer CLI is not installed. Would you like to install it?.");
+    return localize("azure-dev.utils.azd.notInstalled", "Azure Developer CLI is not installed. [Learn More](https://aka.ms/azd-install). Would you like to install it?");
```

Comment:
TODO: Need final text from @savannahostrowski / @gkulin

Reply:
Can we put the learn more link last in the phrase? Otherwise, LGTM.

Reply:
How's this? 
![image](https://user-images.githubusercontent.com/36966225/225066722-7dc2726b-0c95-4658-a746-2cc621421539.png)

Reply:
Yep - ship it!